routes.js

```diff
@@ -50,10 +50,11 @@ export default function getRoutes(
 	{
 		API_SERVER_ROOT_URL,
 		PHOTO_SCALER_SALT,
-	}) {
+	},
+	apiProxyFn$ = apiProxy$) {
```

Comment:
it feels unkosher to have to change the signature of a method in order to support test cases, but i don't know enough about jest to know if there's any way around this.